recipe/meta.yaml

```
@@ -9,7 +9,8 @@ source:
   sha256: c306ec3db5188ee48493cee717667486592e696fb9aa8ed9041c40cf4e3ddfe5
 
 build:
-  noarch: python
+  # notebook <7 isn't available for py>=312 on s390x
+  skip: true  # [s390x]
   number: 0
```

```suggestion
  number: 1
```
Since it was published on services recently.